examples/vision/semisupervised_simclr.py

```diff
@@ -416,19 +417,19 @@ def contrastive_loss(self, projections_1, projections_2):
         # NT-Xent loss (normalized temperature-scaled cross entropy)
 
         # Cosine similarity: the dot product of the l2-normalized feature vectors
-        projections_1 = tf.math.l2_normalize(projections_1, axis=1)
-        projections_2 = tf.math.l2_normalize(projections_2, axis=1)
+        projections_1 = keras.utils.normalize(projections_1, axis=1, order=2)
```

Comment:
You can just use `ops.normalize` with the latest Keras version

Reply:
Yeah. Replaced with Keras ops.